boa/boa-id-v1.html

```diff
@@ -24,15 +24,15 @@ <h4 id="familyName"><a href="#familyName">familyName</a></h4>
 
 <h4 id="additionalName"><a href="#additionalName">additionalName</a></h4>
 
-Any additional name of the BOA to which this ID has been issued (usually used for middle names).
+The additional name(s) of the BOA to which this ID has been issued, if any (usually used for middle names).
 
 <h4 id="idNumber"><a href="#idNumber">idNumber</a></h4>
 
-The identifying BOA number.
+The identifying BOA number as issued by the employer.
```

Comment:
suggestion: "as determined by the employer" or "as asigned by the employer". Don't want to get too confusing with the word "issued" here

Reply:
What would the confusion be exactly? The organization has their own numbering scheme. They are literally in control about who gets what number. Will change, but want to know the nuance

Reply:
It's more a source of truth type thing. Probably not too important at this point but my thought was that saying "issued" here makes it sound like this credential is representing something else that has been issued, rather than being the thing that is issued. While that most likely is the case now, I think it makes sense to treat the digital object as the source of authority.

Looking at it again, since the entity issuig the credential (Justis) is separate from the employer, it may make sense to use "issued" here in reference to a number issued by a different authority than that issuing the credential. In either case "assigned" also makes sense.